packages/settings-library/src/settings_library/aws_s3_cli.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import Field
+
+from .base import BaseCustomSettings
+from .s3 import S3Settings
+
+
+class AwsS3CliSettings(BaseCustomSettings):
+    AWS_S3_CLI_S3: S3Settings = Field(
+        default=None,
+        description="These settings intentionally do not use auto_default_from_env=True because we might want to turn them off if RClone is enabled.",
+    )
```

Comment:
`auto_default_from_env` is used to capture the defaults, that's why you need to turn them off. please remove that part from the comment it's not necessary

```suggestion
    AWS_S3_CLI_S3: S3Settings| None = Field(
        default=None,
        description="These settings intentionally do not use auto_default_from_env=True because we might want to turn them off if RClone is enabled.",
    )
```